packages/kit/test/apps/basics/src/routes/state/url/[slug]/+page.svelte

```diff
@@ -0,0 +1,32 @@
+<script>
+	import { pushState } from '$app/navigation';
+	import { page } from '$app/state';
+	import { untrack } from 'svelte';
+
+	let count = $state(0);
+
+	$effect(() => {
+		page.url;
+		untrack(() => {
+			count += 1;
+		});
+	});
+</script>
+
+<button
+	type="button"
+	class="before"
+	onclick={() => {
+		pushState('', { s: 'test' });
+	}}>slug = before</button
+>
+<button
+	type="button"
+	class="after"
+	onclick={() => {
+		pushState('/state/url/after', { s: 'test' });
```

Comment:
also need a test to pushState with `new URL(...)` to check if it's reactive with a URL instance.